src/main/resources/scripts/install-gdal-databricks.sh

```diff
@@ -27,5 +27,5 @@ cd /usr/lib/python3/dist-packages/osgeo \
 # add pre-build JNI shared object to the path
 # please run MosaicGDAL.copySharedObjects("/dbfs/FileStore/geospatial/mosaic/gdal/") before enabling this init script
 mkdir -p /usr/lib/jni
-cp /dbfs/FileStore/geospatial/mosaic/gdal/libgdalalljni.so /usr/lib/jni
-cp /dbfs/FileStore/geospatial/mosaic/gdal/libgdalalljni.so.30 /usr/lib/jni
\ No newline at end of file
+cp "${MOSAIC_GDAL_JNI_DIR}/libgdalalljni.so" /usr/lib/jni
```

Comment:
@milos-colic shouldn't you have a default value in case that env variable isn't set?  

`FOO="${VARIABLE:=default}"  # If variable not set or null, set it to default.`
^ from [here](https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash)

Also, recommend also add code for `${MOSAIC_GDAL_JNI_DIR}` to be replaced with `initScriptPath` when `setup_gdal` / `prepareEnvironment` are called.

Reply:
I like the idea of the default value. I will add it.

I am not sure what you meant by "Also, recommend also add code for ${MOSAIC_GDAL_JNI_DIR} to be replaced with initScriptPath when setup_gdal / prepareEnvironment are called."

do you mean replace the default value in the script with the initScriptPath? or did you mean something else?

Reply:
I mean when `setup_gdal` is called the actual init script that gets written to the initScriptPath should have its paths changed / written according to the params passed to the function.

Reply:
so you can still override it if you wish using the MOSAIC_GDAL_JNI_DIR variable, and if you dont then it will use default value which is what you supplied to the method that generated the script.

Reply:
makes sense, I see the changes made.